test/test_torch.py

```diff
@@ -11177,7 +11179,7 @@ def test_logical(self, device):
                 torch.ne(x, b, out=boolRes)
                 self.assertEqual(byteRes.bool(), boolRes)
 
-                self.assertEquals(len(warningsCount), 5)
+                self.assertEquals(len(warningsCount), 4)
```

Comment:
This is going to merge conflict with the other comparison op PRs. It is probably better to squash all of the commits together or to use ghstack.